### ISSUE TITLE
IndexError: list index out of range while retrieving highlights

### ISSUE THREAD

**Issue by zweq** [instaloader/instaloader#1348]
**Describe the bug**
Retrieving highlights fails due to IndexError. The issue can be repeated.

**To Reproduce**
instaloader --login zweq --stories --highlights _account_

**Expected behavior**
Highlights are successfully downloaded.

**Error messages and tracebacks**
```
[  3/ 34] <imagefile> exists Traceback (most recent call last):
File "/home/zweq/.local/bin/instaloader", line 10, in <module>
    sys.exit(main())
  File "/home/zweq/.local/lib/python3.7/site-packages/instaloader/__main__.py", line 482, in main
    storyitem_filter_str=args.storyitem_filter)
  File "/home/zweq/.local/lib/python3.7/site-packages/instaloader/__main__.py", line 223, in _main
    fast_update, post_filter, storyitem_filter, latest_stamps=latest_stamps)
  File "/home/zweq/.local/lib/python3.7/site-packages/instaloader/instaloader.py", line 1418, in download_profiles
    self.download_highlights(profile, fast_update=fast_update, storyitem_filter=storyitem_filter)
  File "/home/zweq/.local/lib/python3.7/site-packages/instaloader/instaloader.py", line 80, in call
    return func(instaloader, *args, **kwargs)
  File "/home/zweq/.local/lib/python3.7/site-packages/instaloader/instaloader.py", line 923, in download_highlights
    downloaded = self.download_storyitem(item, highlight_target)
  File "/home/zweq/.local/lib/python3.7/site-packages/instaloader/instaloader.py", line 851, in download_storyitem
    self.download_pic(filename=filename, url=item.video_url, mtime=date_local))
  File "/home/zweq/.local/lib/python3.7/site-packages/instaloader/structures.py", line 1132, in video_url
    version_urls = [self._node['video_resources'][-1]['src']]
IndexError: list index out of range
```

**Instaloader version**
4.8.1

**Comment by aandergr** [comment]
Thanks for reporting! This should be fixed in [Version 4.8.2](https://github.com/instaloader/instaloader/releases/tag/v4.8.2) by #1321.

**Comment by zweq** [comment]
Indeed, the problematic video now only produces a warning. Thanks!
